WordPress/src/test/java/org/wordpress/android/ui/mysite/MySiteViewModelTest.kt

```diff
@@ -190,31 +189,30 @@ class MySiteViewModelTest : BaseUnitTest() {
 
     private val cardsUpdate = MutableLiveData(
             CardsUpdate(
-                    CardsResult(
-                            listOf(
-                                    PostsCardModel(
-                                            hasPublished = true,
-                                            draft = listOf(
-                                                    PostCardModel(
-                                                            id = 1,
-                                                            title = "draft",
-                                                            content = "content",
-                                                            featuredImage = "featuredImage",
-                                                            date = Date()
-                                                    )
-                                            ),
-                                            scheduled = listOf(
-                                                    PostCardModel(
-                                                            id = 2,
-                                                            title = "scheduled",
-                                                            content = "",
-                                                            featuredImage = null,
-                                                            date = Date()
-                                                    )
+                    cards = listOf(
+                            PostsCardModel(
+                                    hasPublished = true,
+                                    draft = listOf(
+                                            PostCardModel(
+                                                    id = 1,
+                                                    title = "draft",
+                                                    content = "content",
+                                                    featuredImage = "featuredImage",
+                                                    date = Date()
+                                            )
+                                    ),
+                                    scheduled = listOf(
+                                            PostCardModel(
+                                                    id = 2,
+                                                    title = "scheduled",
+                                                    content = "",
+                                                    featuredImage = null,
+                                                    date = Date()
                                             )
                                     )
                             )
                     )
+
```

Comment:
Minor (🔍): Consider removing this extra line.

Reply:
In 7d3098c